src/go/rpk/pkg/cli/cmd/redpanda/admin/admin.go

```diff
@@ -63,6 +64,7 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 	cmd.AddCommand(
 		brokers.NewCommand(fs),
 		configcmd.NewCommand(fs),
+		clustercmd.NewCommand(fs),
```

Comment:
Do you think that the command should go under `rpk cluster` rather than `rpk redpanda cluster`. The former has things like maintenance and config and is generally more cluster oriented. Grabbing the cluster health seems to fit into that model.

Here is the rpk help menu which seems to support that view:

```
cluster     Interact with a Redpanda cluster.
redpanda    Interact with a local Redpanda process
```

Reply:
great idea, i am going to move it